client/types/deposits.d.ts

```diff
@@ -52,3 +54,29 @@ export type DepositStatus =
 	| 'in_transit'
 	| 'canceled'
 	| 'failed';
+
+export type PayoutFailureCode =
```

Comment:
See my above comment. This part might be affected if you decide to add the missing enums from https://docs.stripe.com/api/payouts/object#payout_object-failure_code:

- incorrect_account_holder_address
- incorrect_account_holder_tax_id
- invalid_account_number_length

Reply:
Thanks for marking these! I have updated the type def here as well - [072cb46](https://github.com/Automattic/woocommerce-payments/pull/10462/commits/072cb4605b39c70b95de158761c8fd42dbd759e0)